include/inexor/vulkan-renderer/wrapper/texture.hpp

```diff
@@ -65,12 +65,14 @@ class Texture {
     /// @param graphics_card [in] The graphics card.
     /// @param vma_allocator [in] The Vulkan Memory Allocator library handle.
     /// @param texture_data [in] The texture data.
+    /// @param texture_width [in] The width of the texture.
+    /// @param texture_height [in] The height of the texture.
     /// @param texture_size [in] The size of the texture.
     /// @param name [in] The internal memory allocation name of the texture.
     /// @param data_transfer_queue [in] The Vulkan data transfer queue.
     /// @param data_transfer_queue_family_index [in] The queue family index of the data transfer queue to use.
     Texture(const wrapper::Device &device, const VkPhysicalDevice graphics_card, const VmaAllocator vma_allocator,
-            void *texture_data, const std::size_t texture_size, const std::string &name,
+            void *texture_data, const std::uint32_t texture_width, const std::uint32_t texture_height, const std::size_t texture_size, const std::string &name,
```

Comment:
Just a side note:
```Texture &operator=(Texture &&) = default;``` is implicitly deleted.

Reply:
I will review this in anothe PR: https://github.com/inexorgame/vulkan-renderer/issues/209.